### DISCUSSION TITLE
The problem of not being able to deposit to gate.io wallet

### DISCUSSION THREAD

**Discussion by karasgoon** [Chia-Network/chia-blockchain#6843] [Support]
I set the XCH mining reward address to gate.io's wallet. 
17 days ago I mined 2 coins. 
But the reward hasn't come yet. 
Has anyone ever encountered this issue? I want to know the solution.

The problem of not being able to deposit to gate.io wallet

**Comment by loppefaaret** [comment]
Please don't farm directly into an exchange wallet - if the exchange wallet changes your deposit address, anything you get is lost!
farm into a wallet you yourself control

**Reply by computer817** [reply]
Me too, There are ten days, haven't been to the bill！The exchange has been dragged, not to deal with

Hope the official will help us to solve this problem!

**Reply by loppefaaret** [reply]
are you referring to the block that was farmed on block 410486 ? if that is not shown in the account on the exchange wallet, there is nothing we can do about it - it is flagged differently on the chain, as it is not a regular transaction, but a reward... 
so stop farming directly into the exchange wallet

**Reply by computer817** [reply]
Yes!To the extent that the wallet that reward?

**Reply by computer817** [reply]
Exchange is the reply XCH block problem！
Exchange system hasn't been booked for me!

**Reply by loppefaaret** [reply]
there is nothing Chia can do for you - farm into a wallet you control!

**Reply by computer817** [reply]
Reward is to exchange the purse!

**Reply by computer817** [reply]
The problem of not being able to deposit to gate.io wallet
Many miners have met!
Still hope to receive the official help!
thank you

**Reply by loppefaaret** [reply]
what exactly is stopping you from depositing onto gate.io ?
when does the error happen ?
i expect your are able to send off the transaction from the wallet fine...
but it is not showing up on your account on gate.io ?

**Reply by computer817** [reply]
Yes, there is no display on the account
gate.io！reply Block problem!

**Reply by loppefaaret** [reply]
if you look up the transaction on a block explorer like chiaexplorer.com or xchscan.com - do you see the transaction ?

**Reply by computer817** [reply]
Query! Is to be found

**Reply by loppefaaret** [reply]
if you can find it on chiaexplorer.com or xchscan.com - then there isn't anything Chia can do for you, you completed a successful transaction!

**Reply by computer817** [reply]
Yes, can query to! Exchange means XCH problem, so hope I can with official GATE. IO communication under the problem, help us to finish as soon as possible to the account
thank you

**Reply by loppefaaret** [reply]
what specific transaction are you refering to ? i need something before i can do anything

**Reply by computer817** [reply]
I set the XCH mining reward address to gate.io's wallet.
17 days ago I mined 2 coins.
But the reward hasn't come yet
gate.io Was it a matter of chia caused! Didn't say specific reason!
So I hope the official can communicate with them!

**Reply by loppefaaret** [reply]
no! do not farm directly into an exchange wallet, we cannot help with that! that is the exchange decision - rewards from farming are different than transactions - it is up to gate.io to decide what they do with it!
always farm into a wallet that _you_ control!
as far as Chia Network is concerned, it is clearly shown that the reward from farming that block, has gone to the address you have now removed from the post - there is nothing we can do here...
if you dont have the private key to the wallet, that is _your_ problem, you decided to send your farming rewards to another wallet that you do not control
all you can do is ask gate.io to help, since - as far as you say - is them that control the wallet, and it is their wallet that received the reward from that block win

**Comment by computer817** [comment]
Understand! Now have stopped, but two XCH, no to the account!
Chia project, whether with GATE. IO communication channels! Help us to coordinate!
Now, GATE IO hasn't been very good to deal with this problem! Has not arrive!

**Reply by loppefaaret** [reply]
i consider this subject closed at this point, if gate.io has problems, they should be aware of where to get a hold of the Chia Network team. How they manage their wallets is up to them!

**Comment by BrandtH22** [comment]
@karasgoon , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
